network/src/commonMain/kotlin/com/wire/kalium/network/HttpClientProvider.kt

```diff
@@ -51,6 +52,7 @@ internal fun provideBaseHttpClient(
             level = LogLevel.ALL
         }
     }
+    install(WebSockets)
```

Comment:
Wouldn't this cause some sort of exception on iOS targets as Ktor only support Websockets for Android and JVM? For now it is not a problem at all, but maybe we should write some comments and think on how to inject these websockets on a platform targetted mode (perhaps using the expect-actual KMP way?).

Reply:
Yes!
As soon as we add `iOS` as a target, this will break.

I had some ideas about creating an `expect class WebsocketClient` which allows us to pass an URL and get an `incoming` flow.
Then we can implement per platform (most will just use the current one), but iOS for example would have its own magic.

I'll add a comment for now.